orchestrator/api/helpers.py

```diff
@@ -257,7 +257,7 @@ def enrich_process(p: ProcessTable) -> _ProcessListItem:
         last_status=p.last_status,
         last_step=p.last_step,
         subscriptions=subscriptions,
-        workflow=p.workflow_name,
+        workflow=str(p.workflow_name),
```

Comment:
Why do you need to cast it to str?

Reply:
Mypy will see it as a Column type otherwise.

Reply:
oke